cmd/ztest/ztest.c

```diff
@@ -6431,30 +6430,31 @@ ztest_check_path(char *path)
 }
 
 static void
-ztest_get_zdb_bin(char *bin, int len)
+ztest_get_binary(const char *name, const char *env_var, char *bin, int len)
 {
-	char *zdb_path;
+	char *env_path;
 	/*
-	 * Try to use ZDB_PATH and in-tree zdb path. If not successful, just
-	 * let popen to search through PATH.
+	 * Try to use env_var and in-tree paths. If not successful, just
+	 * allow popen to search through PATH.
 	 */
-	if ((zdb_path = getenv("ZDB_PATH"))) {
```

Comment:
I believe  `ZDB_PATH` is entirely unused and the comment here is misleading.  The logic below is used to determine if `ztest` in running in-tree.  This might be a good opportunity to retire `ZDB_PATH` which is entirely undocumented.

Reply:
OK, retired `ZDB_PATH`